Set-1/Solved Programs/BrainCar/BrainCar.java

```diff
@@ -0,0 +1,74 @@
+package com.braincar;
+
+import java.util.logging.Logger;
+
+public class BrainCar {
+	
+	private static int eachconsecutiveCount=1;
+	private static int consecutiveDifferentVar[]=new int[3];				
+	private static int consecutiveRepeatCount=0;
+	
+	
+	/**
+	 * Checking BrainCarName
+	 *
+	 * @param name of the car
+	 * @return OK or Not OK
+	 */
+	public static String checkBrainCarName(String name) {
+		
+		int len=name.length();
+				
+				
+				/** The logger. */
+			    Logger logger = Logger.getLogger(BrainCar.class.getName());

```

Comment:
logger defined inside a method?

Reply:
No its not good inside the method . i will move to outside the method.